mobile/ios/scripts/build.sh

```diff
@@ -1,9 +1,22 @@
 #!/bin/sh -e
+ROOTDIR=$(cd `dirname "${BASH_SOURCE[0]}"` && pwd -P)
 
-./scripts/build_arch.sh iPhoneSimulator i386
-./scripts/build_arch.sh iPhoneSimulator x86_64
-./scripts/build_arch.sh iPhoneOS armv7
-./scripts/build_arch.sh iPhoneOS armv7s
-./scripts/build_arch.sh iPhoneOS arm64
+if [ $# -gt 1 ]; then
+    ARCHS=$@
+else
+    ARCHS="i386 x86_64 armv7 armv7s arm64"
+fi
 
-./scripts/build_frameworks.sh
+echo "Building for this architectures: $ARCHS"
+
+for ARCH in ${ARCHS}
+do
```

Comment:
Please use: ```for ...; do``` rather than putting do in a newline. Vertical screen estate is precious.